packages/quantic/cypress/integration/breadcrumb-manager/breadcrumb-manager.cypress.ts

```diff
@@ -25,8 +32,12 @@ describe('quantic-breadcrumb-manager', () => {
     interceptSearch();
     cy.visit(breadcrumbManagertUrl);
     configure(options);
+    if (options.useCase !== useCaseEnum.search) {
+      cy.wait(1000);
```

Comment:
I am not waiting for an element, I am waiting for this `window.coveoHeadless[engineId]?.bundle`

Reply:
Well, once the component under test is initialized, I guess we could check for the existence of a specific element or CSS class to know it is initialized, right? If we can find a reliable way to know the component is initialized, the test would be faster (because it should take less than 1sec to initialize) and be more reliable (because Cypress default timeout is 4sec in case it takes longer to initialize).

Reply:
I did the change